frontend/src/hooks/useFileExport.ts

```diff
@@ -0,0 +1,84 @@
+import axios from "axios";
+import { useSnackbar } from "notistack";
+import { useCallback } from "react";
+import { useSelector } from "react-redux";
+import { selectDocument } from "../store/documentSlice";
+import { selectEditor } from "../store/editorSlice";
+
+export const enum FileExtension {
+	Markdown = "markdown",
+	HTML = "html",
+	PDF = "pdf",
+}
+
+interface UseFileExportReturn {
+	handleExportToPDF: () => void;
+	handleExportToHTML: () => void;
+	handleExportToMarkdown: () => void;
+}
+
+export const useFileExport = (): UseFileExportReturn => {
+	const { enqueueSnackbar } = useSnackbar();
+	const editorStore = useSelector(selectEditor);
+	const documentStore = useSelector(selectDocument);
+
+	const markdown = editorStore.doc?.getRoot().content?.toString() || "";
+	const documentName = documentStore.data?.title || "codepair_document";
+
+	const handleExportFile = useCallback(
+		async (exportType: string) => {
+			try {
+				enqueueSnackbar(`${exportType.toUpperCase()} 파일 내보내기 시작...`, {
+					variant: "info",
+				});
+
+				const response = await axios.post(
+					"/files/export-markdown",
+					{
+						exportType,
+						content: markdown,
+						fileName: documentName,
+					},
+					{
+						responseType: "blob",
+						headers: {
+							Accept: "application/octet-stream",
+						},
+					}
+				);
```

Comment:
Please move it to `frontend/src/hooks/api/file.ts`.
And the all API calls in CodePair uses `react-query`.

Reply:
Fix useFileExport-related changes in [this commit](https://github.com/yorkie-team/codepair/pull/238/commits/9f0a7d7986d3e20153e52ba22f43a74f3ff2fba4).